src_c/math.c

```diff
@@ -756,7 +765,10 @@ vector_inplace_floor_div(pgVector *o1, PyObject *o2)
 static PyObject *
 vector_neg(pgVector *self)
 {
-    pgVector *ret = (pgVector *)pgVector_NEW(self->dim);
+    pgVector *ret;
+
+    ret = (pgVector *)_vector_subtype_new(Py_TYPE(self), self->dim);
```

Comment:
minor code appearance nitpick, this bit of code will look neater if it's expressed in one line, like 
` pgVector *ret  = (pgVector *)_vector_subtype_new(Py_TYPE(self), self->dim);`
Here, and also in all functions below